community/modules/utils/identity/README.md

```diff
@@ -0,0 +1,36 @@
+## Description
+
+This module does not do anything, it just returns the value passed to it.
+It creates no resources and uses no providers.
```

Comment:
Since we will not use this in example blueprints, please document what it is to be used for. What would keep me from deleting this as dead code as soon as it is submitted.